frontend/lib/src/components/shared/Tooltip/OverflowTooltip.test.tsx

```diff
@@ -47,7 +48,12 @@ describe("Tooltip component", () => {
         style={{}}
       >
         the child
-      </OverflowTooltip>
+      </OverflowTooltip>,
```

Comment:
what change required this thing to now need to be wrapped in the theme provider? 🤔

Reply:
Oh, this is just a weirdly written test since it mocks `React.*` methods. That is not a recommended way to mock things since it's too broad and leads to unexpected coupling in tests like this one. I could have rewritten this test but I took the easier way out by giving this a specific wrapper instead since it was the only test that was affected by this.